src/Protocol/ActivityPub/Processor.php

```diff
@@ -1206,6 +1206,52 @@ public static function deletePerson($activity)
 		Logger::info('Deleted contact', ['object' => $activity['object_id']]);
 	}
 
+	/**
+	 * Blocks the user by the contact
+	 *
+	 * @param array $activity
+	 * @throws \Exception
+	 */
+	public static function blockPerson($activity)
```

Comment:
```suggestion
	public static function blockUser($activity)
```

Reply:
Since the action target is in the "Account" array (see `ACCOUNT_TYPES`), it's better to change it to `blockAccount`.